addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -294,6 +310,10 @@ registerModel({
         compact: attr({
             related: 'threadViewer.compact',
         }),
+        /**
+         * States the OWL component of this thread view.
+         */
+        component: attr(),
```

Comment:
Maybe I missed somewhere, but component of thread view does not look used anymore after recent changes?

Reply:
I think you're right, no longer used. Will do clean up